README.md

```diff
@@ -33,21 +33,19 @@ Default conda channel:
 * [`jupyterlab`](https://jupyterlab.readthedocs.io/en/stable/)
 * [`nglview`](http://nglviewer.org/nglview/latest/)
 
-#### Known Bugs
```

Comment:
nglview works on windows WSL so I'm good with saying it works (not going to have the time/effort to test it directly on the windows side of things).